webhook/server.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const https = require('https');
+const fs = require('fs');
+const path = require('path');
+
+const app = express()
+const port = 3000;
+
+const options = {
+  key: fs.readFileSync("cert/server.key"),
+  cert: fs.readFileSync("cert/server.cert"),
+};
+
+const functions = require('./indexFunctions');
+
+app.get('/', (req, res) => {
+  res.send("Hello world");
+});
+
+app.use(express.static('public'));
+
+app.get('/webhook', (req, res) => {
```

Comment:
Who will call that webhook?

Reply:
Zendesk itself will call it when a ticket is updated.  It passes in the ticketId

Reply:
true..I can update the docs with screenshots and the like